lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -553,25 +564,17 @@ LogicalResult iterativelyFuseProducerAndConsumerOfTiledOp(
 ///       yield %5
 ///   }
 /// }
-static LogicalResult isSelfTiledOp(Operation *targetOp) {
-  // 0. check tilable
-  if (!isa<TilingInterface>(targetOp))
+static LogicalResult isTiledOpInLoop(Operation *targetOp) {
+  // 1. check tilable
```

Comment:
The limitation of single tiled op inside loop has been removed now. @kurapov-peter Please help to take a look :).